person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(id, age: nil, parent_permission: true, name: 'Unknown')
+    @id = id

```

Comment:
- [ ] [optional} I would suggest instead of just using the id variable you generate a unique random if for each person created using the Random inbuilt method eg  `@id = Random.rand(1..10000)` for id number in the range between 1 and 10000 you can choose your own range.